public/index.html

```diff
@@ -112,6 +113,20 @@ <h2 data-i18n="titlebox.resources.title"></h2>
       <!-- the Leaflet map container -->
       <div id="map"></div>
 
+      <!--The search bar-->
+      <form id="search-bar-form">
+        <div id="search-bar-div">
+          <input type = "text"
+                 name="search"
+                 id="search-bar"
+                 aria-label="search"
+                 value = "&#128269; &nbsp; &nbsp; Enter Nation, state, province, city or zipcode"
```

Comment:
@jimboweb to keep things consistent perhaps you could change `Nation` to `nation`.

Reply:
![image](https://user-images.githubusercontent.com/10737214/86396631-29578680-bc57-11ea-87e3-19a9f51e61b1.png)

Reply:
changed the capitalization